man/assocTestAggregate.Rd

```diff
@@ -114,7 +114,11 @@
     \item{pval.method}{The p-value calculation method used. When standard SKAT is used, one of "integration" or "saddlepoint"; when fastSKAT random projections are used to approximate eigenvalues of the genotype covariance matrix, one of "ssvd_integration" or "ssvd_saddlepoint"; when fastSKAT random projections are used to approximate both the eigenvalues and the trace of the genotype covariance matrix, one of "rsvd_integration" or "rsvd_saddlepoint".}
 
     If \code{test} is \code{"SMMAT"} or \code{"fastSMMAT"}:
+    \item{Score_burden}{The value of the score function for the burden test}
+    \item{Score.SE_burden}{The estimated standard error of the Score for the burden test}
+    \item{Stat_burden}{The score Z test statistic for the burden test}
     \item{pval_burden}{The burden test p-value}
+    \item{Q_theta}{The SMMAT test statistic}
```

Comment:
@amstilp Q_theta: The test statistic for the adjusted SKAT test (which is asymptotically independent of the burden test)

Reply:
Updated